ariba/2.14.6/Dockerfile

```diff
@@ -0,0 +1,82 @@
+FROM ubuntu:bionic as app
+
+LABEL base.image="ubuntu:bionic"
+LABEL dockerfile.version="2"
+LABEL software="ARIBA"
+LABEL software.version="2.14.6"
+LABEL description="ARIBA: Antimicrobial Resistance Identification By Assembly"
+LABEL website="https://github.com/sanger-pathogens/ariba"
+LABEL license="https://github.com/sanger-pathogens/ariba/blob/master/LICENSE"
+LABEL maintainer="Curtis Kapsak"
+LABEL maintainer.email="pjx8@cdc.gov"
+LABEL maintainer2="Harry Hung"
+LABEL maintainer2.email="ch31@sanger.ac.uk"
+
+# prevents having to enter commands during apt-get install
+ENV DEBIAN_FRONTEND=noninteractive
```

Comment:
Can this ENV be changed to an ARG? ENV persist, and we try not to include any of those if we don't have to.